docs/cudf/source/conf.py

```diff
@@ -202,14 +208,51 @@
 autoclass_content = "init"
 
 # Config AutoStructify
-github_doc_root = 'https://github.com/rtfd/recommonmark/tree/master/doc/'
+github_doc_root = "https://github.com/rtfd/recommonmark/tree/master/doc/"
+
+# Replace API shorthands with fullname
+reftarget_aliases = {
```

Comment:
Are `reftarget_aliases` and `internal_names_to_ignore` sphinx config vars? If not, should we prefix them with `_` or something to indicate that?